src/kOS.Safe/Encapsulation/StringValue.cs

```diff
@@ -216,6 +271,7 @@ private void StringInitializeSuffixes()
             AddSuffix("TRIM",       new NoArgsSuffix<StringValue>(() => Trim()));
             AddSuffix("TRIMEND",    new NoArgsSuffix<StringValue>(() => TrimEnd()));
             AddSuffix("TRIMSTART",  new NoArgsSuffix<StringValue>(() => TrimStart()));
+            AddSuffix("TONUMBER",   new VarArgsSuffix<ScalarValue, Structure>(ToScalarVarArgsWrapper));
```

Comment:
*shrug* : It's all the same to me - we could do it either way.  I could change it.